pressbooks/pressbooks
# Add about the authors information to XHTML export routine

Related issue: https://github.com/pressbooks/pressbooks/issues/2347

This PR adds the **About the Authors** section at the end of each chapter for XHTML and EPUB (using [the pb-books PR](https://github.com/pressbooks/pressbooks-book/pull/853)) exports, including PDFs through a new method that adds the HTML at the end of the string of each chapter .

This PR requires pressbooks-book change, see: https://github.com/pressbooks/pressbooks-book/pull/853

### Test
- For a random book, assign different authors to a different chapters. Make sure those authors are book's contributor in the Book Info section.
- Make sure Appearance > Theme Options > About the authors options is enabled.
- To test it better, you may add picture, url, description and/or other fields to the contributors in Book Info > Contributors section.
- Export the book to EPUB, PDF and XHTML and make sure the authors are added at the end of those chapter with authors.
